modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCenter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.credential;
+
+import java.util.List;
+import org.eclipse.xpanse.modules.models.enums.Csp;
+
+/**
+ * The credential center.
+ */
+public class CredentialCenter {
+
+    /**
+     * Get the credential abilities of Csp.
+     *
+     * @param csp The csp to get.
+     */
+    List<Credential> getCredentialAbility(Csp csp) {

```

Comment:
should the return type be List<CredentialAbility>?

Reply:
No, the return type is not a problem, this method designed to get the detailed Credential abilities (e.g., with variables for `CredentialInput`)